src/Configuration/Schema/SchemaConfigurationFactory.php

```diff
@@ -59,6 +59,9 @@ public function create(string $path, stdClass $rawConfig): SchemaConfiguration
             self::createLogs($rawConfig->logs ?? new stdClass()),
             self::normalizeString($rawConfig->tmpDir ?? null),
             self::createPhpUnit($rawConfig->phpUnit ?? new stdClass()),
+            $rawConfig->ignoreMsiWithNoMutations ?? null,
+            $rawConfig->minMsi ?? null,
+            $rawConfig->minCoveredMsi ?? null,
```

Comment:
I think you are missing a `(float)` here and up

Reply:
👍